src/hott/06-contractible.rzk.md

```diff
@@ -110,69 +110,71 @@ A type is contractible if and only if its terminal map is an equivalence.
 
 ```rzk
 #def terminal-map-is-equiv
-  (A : U)
+  ( A : U)
   : U
   := is-equiv A Unit (terminal-map A)
 
 #def contr-implies-terminal-map-is-equiv-retr
-  (A : U)
-  (AisContr : is-contr A)
+  ( A : U)
+  ( A-is-contr : is-contr A)
   : has-retraction A Unit (terminal-map A)
   :=
-    ( constant Unit A (contraction-center A AisContr) ,
-      \ y -> (contracting-htpy A AisContr) y)
+    ( constant Unit A (contraction-center A A-is-contr) ,
+      \ y -> (contracting-htpy A A-is-contr) y)
 
 #def contr-implies-terminal-map-is-equiv-sec
-  (A : U)
-  (AisContr : is-contr A)
+  ( A : U)
+  ( A-is-contr : is-contr A)
   : has-section A Unit (terminal-map A)
-  := (constant Unit A (contraction-center A AisContr) , \ z -> refl)
+  := ( constant Unit A (contraction-center A A-is-contr) , \ z -> refl)
 
 #def contr-implies-terminal-map-is-equiv
-  (A : U)
-  (AisContr : is-contr A)
+  ( A : U)
+  ( A-is-contr : is-contr A)
   : is-equiv A Unit (terminal-map A)
   :=
-    ( contr-implies-terminal-map-is-equiv-retr A AisContr ,
-      contr-implies-terminal-map-is-equiv-sec A AisContr)
+    ( contr-implies-terminal-map-is-equiv-retr A A-is-contr ,
+      contr-implies-terminal-map-is-equiv-sec A A-is-contr)
 
 #def terminal-map-is-equiv-implies-contr
-  (A : U)
+  ( A : U)
   (e : terminal-map-is-equiv A)
   : is-contr A
-  := ((first (first e)) unit , (second (first e)))
+  := ( (first (first e)) unit ,
+       (second (first e)))
 
 #def contr-iff-terminal-map-is-equiv
-  (A : U)
+  ( A : U)
   : iff (is-contr A) (terminal-map-is-equiv A)
   :=
-    ( (contr-implies-terminal-map-is-equiv A) ,
-      (terminal-map-is-equiv-implies-contr A))
+    ( ( contr-implies-terminal-map-is-equiv A) ,
+      ( terminal-map-is-equiv-implies-contr A))
 
 #def equiv-with-contractible-domain-implies-contractible-codomain
-  (A B : U)
-  (f : Equiv A B)
-  (is-contr-A : is-contr A)
+  ( A B : U)
+  ( f : Equiv A B)
+  ( is-contr-A : is-contr A)
   : is-contr B
-  := (terminal-map-is-equiv-implies-contr B
-      (second
-      (comp-equiv B A Unit
-        (inv-equiv A B f)
-        (terminal-map A , (contr-implies-terminal-map-is-equiv A is-contr-A)))))
+  := ( terminal-map-is-equiv-implies-contr B
+      ( second
+        ( comp-equiv B A Unit
+          ( inv-equiv A B f)
+          ( terminal-map A ,
+            ( contr-implies-terminal-map-is-equiv A is-contr-A)))))
```

Comment:
When you parenthesize one argument of a binary operator like this, I also tend to parenthesize the other. I.e. here:
```rzk
          ( ( terminal-map A) ,
            ( contr-implies-terminal-map-is-equiv A is-contr-A)))))
```
This presents the arguments on even footings, and more clearly displays the tree structure.